builder/vmware/common/driver_fusion5.go

```diff
@@ -139,6 +157,32 @@ func (d *Fusion5Driver) Verify() error {
 		return err
 	}
 
+	libpath := filepath.Join("Library", "Preferences", "VMware Fusion")
```

Comment:
When I build, I get an issue because libpath isn't an absolute path (no `/` before `Library`).  Also, this path seems OSX specific.  @mwhooker is this code supposed to be generalizable to linux and windows?